### DISCUSSION TITLE
posts/autonomous-4/

### DISCUSSION THREAD

**Discussion by giscus** [seyeon040768/seyeon040768.github.io#11] [Comments]
# posts/autonomous-4/

참고사항

https://blog.seyeon-lee.com/posts/autonomous-4/

**Comment by minmaxHong** [comment]
아무래도, 카메라가 고정이 되어있으면 tracking하는데는 물체가 움직이는 것을 잘 검출할 수 있지만
자동차에서 1인칭으로 바라볼 때, tracking을 할 때는 상대 속도를 구하기에는 어려운 점이 있을 것 같은데, 이를 어떻게 해결할건지?

개인적인 생각으로는 현재 자신이 타고 있는 자동차의 속도와 다른 object의 속도도 비교해야할거같은데.. 이는 어떻게 하실것인지 궁금합니다.

**Reply by jongmyeongpark** [reply]
"계"를 이동시키면 해결될 거 같습니다. golbal좌표계로 이동시켜서 자동차 자신의 글로벌 좌표값들을 각 대상에 대해서 뺀다면 카메라가 고정되어있는 상황과 같은 상황이 됩니다.

**Comment by jongmyeongpark** [comment]
라이다 없이 카메라만으로 위치를 파악 할 수 있는지 궁금합니다. 이런 영상처리 기법 같은게 있는지 불가능해 보이지는 않을 거 같아서 더 궁금합니다. 감사합니당
